### PR TITLE
chore: disable playwright retries set up

### PR DESCRIPTION
**Pull request by gre**
### 📝 Description

LLD e2e tests has been recently flaky and unfortunately by design: we have used `retries: process.env.CI ? 2 : 0` playwright suggested configuration which overall is a bad idea if we don't closely monitor what gets "flaky" over the time (and we didn't).

So in `playwright.config.ts`, `retries: 2`basically told playwright "it's fine if a tests fails 2/3 of the time". In our case, a mismatch screenshot will be allowed to run 2 more times until it really fails. This contributed to introduce even more flaky cases in the past months because it's really hiding test issues under the carpet. 

We reached a point where there were so much flaky tests that we had more chance to fail one of the flaky tests that succeed, even with the retries in place. It also contributed to make the CI runs slower since we often need to retry.

This PR has helped identifying and fixing many of the flaky tests and can be merged once we assume there are no major flaky test anymore. 

For each flaky case we detected through this PR, we used this PR as a testing sandbox and once the fix was confirmed, we sent a side PR that was sending the fix alone. At the end, this PR will only contain the configuration changes. 

- [ ] most of the flaky tests has been fixed. (when this checkbox is checked, we are ready to merge)

### ❓ Context

- **Impacted projects**: LLD tests 
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-9607 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Review by Josemoe — COMMENTED** [review]
I believe I checked my file

**Comment by gre** [comment]
Here, we have run **16 runs** of LLD e2e tests without the "retries" mecanism (by pushing an empty commit every 1 hour) and here is the results. Overall the results are pretty good considering we remove any retry mecanism here. Also there were no new "surprise" topics so we are confident being able to go develop (and skip some if necessary).

Success rates:

- Most of them have timeout on windows because job was always cancelled by my 1-hour loop. so we ignore windows here. next time I do this i will try to always wait for previous run before starting a new one.
- Linux success rate: 13/16
- Mac success rate: 9/16

Causes of failures
- 3x : `settings/localization.spec.ts:12:5 › Settings › go to settings -> change language with device in English. Error: Screenshot comparison failed: await expect(page).toHaveScreenshot("settings-français-with-device-l10n.png");`
- 4x : LIVE-9623. device actions timeout. -> fixed by #5247 
- 4x : liveapp-sdk
  - 1x : `services\buy.spec.ts:41:5 › Buy / Sell @smoke: apps\ledger-live-desktop\tests\models\LiveAppWebview.ts#L120. timeout`
- 2x : Wallet API: transaction.sign
- 1x : LIVE-10015. macos instance issue with no such file or directory error.

File: apps/ledger-live-desktop/tests/models/LiveAppWebview.ts [hunk 1]
```diff
@@ -120,12 +120,7 @@ export class LiveAppWebview {
     const result: boolean = await this.page.evaluate(textToCheck => {
       const webview = document.querySelector("webview");
       return (webview as WebviewTag)
-        .executeJavaScript(
-          `(function() {
-        return document.querySelector('*').innerHTML;
-      })();
-    `,
-        )
+        .executeJavaScript("document.body.innerHTML")
```
Comment (Justkant) [inline apps/ledger-live-desktop/tests/models/LiveAppWebview.ts:123]: Thx looks cleaner 🙏

**Review by Justkant — APPROVED** [review]
Thx

**Comment by gre** [comment]
we ran the test 36 times. (12 runs on 3 OS)
- 26 success.
- 8 errors related to our CI (cancelled/job not started)
- 2 errors to dig.

https://github.com/LedgerHQ/ledger-live/actions/runs/6736421809

➡️  edge case due to not defining a timeout on openApp() in case the loader don't appear anyway. fixing by https://github.com/LedgerHQ/ledger-live/pull/5299

https://github.com/LedgerHQ/ledger-live/actions/runs/6737960307

```
  1) services/liveapp-sdk.spec.ts:34:5 › Live App SDK methods @smoke › Verify Address - modal ──────
    Error: Screenshot comparison failed:

      37910 pixels (ratio 0.05 of all image pixels) are different.

    Expected: /Users/admin/actions-runner/_work/ledger-live/ledger-live/apps/ledger-live-desktop/tests/specs/services/liveapp-sdk.spec.ts-snapshots/live-app-verify-address-darwin.png
    Received: /Users/admin/actions-runner/_work/ledger-live/ledger-live/apps/ledger-live-desktop/tests/artifacts/test-results/services-liveapp-sdk-Live-App-SDK-methods-smoke/live-app-verify-address-actual.png
        Diff: /Users/admin/actions-runner/_work/ledger-live/ledger-live/apps/ledger-live-desktop/tests/artifacts/test-results/services-liveapp-sdk-Live-App-SDK-methods-smoke/live-app-verify-address-diff.png

    Call log:
      - page._expectScreenshot with timeout 41000ms
      -   verifying given screenshot expectation
      - taking page screenshot
      -   disabled all CSS animations
      - 37910 pixels (ratio 0.05 of all image pixels) are different.
      - waiting 100ms before taking screenshot
      - taking page screenshot
      -   disabled all CSS animations
      - captured a stable screenshot
      - 37910 pixels (ratio 0.05 of all image pixels) are different.


      83 |     await liveAppWebview.verifyAddress();
      84 |     await deviceAction.openApp();
    > 85 |     await expect.soft(page).toHaveScreenshot("live-app-verify-address.png");
         |                             ^
      86 |   });
```

investigating...

**Review by Justkant — APPROVED** [review]

**Comment by nabil-brn** [comment]
All part of code I've changed in [this commit (8419afe)](https://github.com/LedgerHQ/ledger-live/pull/4918/commits/8419afee66efa84ec6113d66a224ac26eede225f) generated the right screenshots but during the update that [one](https://github.com/LedgerHQ/ledger-live/pull/4918/files#diff-4454a5641d93504596cac2802dd4154cd6a4f6c6cefbbef8861ddb84a94bbfbd) changed for no reason and introduced a wrong reference for comparison. I'll try to make it less flaky, this is the kind of stuff that frustrate developers and I can understand it.

**Review by valpinkman — APPROVED** [review]
